ouroboros-consensus-cardano/test/cardano-test/Test/Consensus/Cardano/Translation.hs

```diff
@@ -204,32 +219,47 @@ PCons
         (CardanoEras Crypto)
     tls = translateLedgerState hardForkEraTranslation
 
-babbageToConwayLedgerStateTranslation ::
+conwayToDijkstraLedgerStateTranslation ::
   RequiringBoth
     WrapLedgerConfig
     TranslateLedgerState
-    (ShelleyBlock (Praos Crypto) BabbageEra)
     (ShelleyBlock (Praos Crypto) ConwayEra)
-babbageToConwayLedgerStateTranslation = translateLedgerStateBabbageToConwayWrapper
+    (ShelleyBlock (Praos Crypto) DijkstraEra)
+conwayToDijkstraLedgerStateTranslation = translateLedgerStateConwayToDijkstraWrapper
 
 -- | Tech debt: The babbage to conway translation performs a tick, and we would
 -- need to create a reasonable ledger state. Instead this is just a copy-paste
 -- of the code without the tick.
 --
 -- This should be fixed once the real translation is fixed.
-translateLedgerStateBabbageToConwayWrapper ::
+-- translateLedgerStateBabbageToConwayWrapper ::
```

Comment:
> Tech debt: The babbage to conway translation performs a tick, and we would need to create a reasonable ledger state. Instead this is just a copy-paste of the code without the tick.

Ah, this has actually been resolved in #1297 even before UTxO HD got merged; so this can definitely be removed.